packages/amplify-category-function/provider-utils/awscloudformation/utils/invoke.js

```diff
@@ -0,0 +1,67 @@
+const path = require('path');
+const fs = require('fs');
+
+function loadFunction(fileName) {
+  return require(path.resolve(fileName));
+}
+/*
+var options = this.options({
+    'packageFolder': './',
+    'handler': 'handler',
+    'fileName': 'index.js',
+    'event': 'event.json'
+});
+*/
+function invokeFunction(options) {
+  console.log('Testing function locally');
+  let cwd;
+  if (options.packageFolder) {
+    cwd = process.cwd();
+    process.chdir(path.resolve(options.packageFolder));
+  }
+
+  const context = {
```

Comment:
Could we allow context to be passed as part of options to allow custom implementation of context functions.